### DISCUSSION TITLE
where does tubeup download the video files to?

### DISCUSSION THREAD

**Discussion by 6xolotl** [bibanon/tubeup#325] [Q&A] [answered]
i cant find it on my own, im not that good with computers 💀

**Comment by mrpapersonic** [comment, marked as answer]
`%USERPROFILE%\.tubeup\downloads` on Windows or `~/.tubeup/downloads` everywhere else

**Reply by 6xolotl** [reply]
i went to %USERPROFILE% and theres no .tubeup folder… even manually typing in the file address didnt do anything, it just gave me an error.

**Reply by mrpapersonic** [reply]
if you're trying to find videos tubeup has uploaded already, they get deleted automatically once the item is uploaded to IA

**Reply by 6xolotl** [reply]
i cancelled the download almost immediately after it started, but from looking at my harddrive space, it looks like the file still exists somewhere on the drive. searching for it by name doesnt give me anything, and the folder address this person gave me doesnt appear to exist on my computer.

**Comment by 6xolotl** [comment]
why is this marked as answered??? i didnt solve my problem????? 💀

**Reply by vxbinaca** [reply]
you asked where the location of the folder is and he gave it to you.

**Reply by 6xolotl** [reply]
and it didnt work… because the folder literally doesnt exist on my device… so theres still an issue of a rogue file somewhere on my drive…

**Reply by vxbinaca** [reply]
if you've uploaded things to IA before then the folder exists.

**Reply by 6xolotl** [reply]
sorry, just saw this now, i never check my email.

i havent uploaded anything, thats the problem. i got partway through the download and then changed my mind, but the space the file took up didnt disappear off my harddrive, so the thing is somewhere but i dont know where. do i need to upload something else to reveal the file??

**Reply by vxbinaca** [reply]
How are you using Tubeup?

**Reply by vxbinaca** [reply]
Also stop unmarking the answer as the answer because it's the correct answer.

**Reply by 6xolotl** [reply]
i unmarked it like once…
also i was going to use it then changed my mind after a download had already started, but the drive space that the partially-downloaded file took up didnt go away when i closed the application and i couldnt find it by searching for the filename, so i opted to ask here what the deal with that is.
im just gonna try to upload something else and see if that fixes the problem of the ghost folder :/
